jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -52,7 +52,9 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     cubeviz_helper.app.add_data_to_viewer('flux-viewer', 'moment 0')
 
     result = dc[1].get_object(cls=CCDData)
-    assert result.shape == (4, 2)  # Cube shape is (2, 2, 4)
+    # This is (ny, nx), see _create_spectrum1d_cube_with_fluxunit for explanation
+    # about what goes where.
+    assert result.shape == (2, 4)
```

Comment:
This is a result of removing `flux = np.moveaxis(flux, 1, 0)` in Cubeviz parser. I think the new value is correct and I try to explain why here.